### PR TITLE
Introduce quarkus-file-vault

### PR DESCRIPTION
**Pull request by gastaldi**
/cc @sberyozkin

### CODE REVIEW

File: quarkus-file-vault.tf [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Create repository
+resource "github_repository" "quarkus_file_vault" {
+  name                   = "quarkus-file-vault"
+  description            = "A cool description"
```
Comment (gastaldi) [inline quarkus-file-vault.tf:4]: Go ahead and add a description here

Comment (sberyozkin) [reply quarkus-file-vault.tf:4]: I propose `Credentials Provider which extracts secrets from Java KeyStore` for a start.

Comment (gastaldi) [reply quarkus-file-vault.tf:4]: `A Vault CredentialsProvider implementation which extracts secrets from Java KeyStore` ?

**Review by sberyozkin — APPROVED** [review]
LGTM, thanks @gastaldi

**Pull request merged**